docs/src/pages/start/quasar-cli.md

```diff
@@ -61,6 +61,8 @@ Pick `Quasar CLI with Vite` if you want:
     $ npm init quasar
     ```
     <br>
+    
+    If you are using npm, you will be prompted to install 'create-quasar' and asking if it is 'Ok to proceed?' Press enter to confirm and install this required package.
```

Comment:
It's not just about `npm`, it's a standard package manager behavior. And, it doesn't ask it if it's cached, which will not be the case for the first time but might be for consequent tries, so using "may" instead of "will" will clarify that. Also, let's put it inside a tip box.
```suggestion
    ::: tip
    You may be presented with a confirmation to install the `create-quasar` package, press the enter key to confirm.
    :::
```